### ISSUE TITLE
esc_2x2 giving different results than Wilson's

### ISSUE THREAD

**Issue by djmeter** [strengejacke/esc#9]
I ran the following code and got a different result than with the web-based 'Practical Meta-Analysis Effect Size
Calculator' from David B. Wilson (<http://www.campbellcollaboration.org/escalc/html/EffectSizeCalculator-Home.php>)

esc_2x2(
  grp1yes = 205, 
  grp1no = 98,   
  grp2yes = 512, 
  grp2no = 178,  
  es.type = c("r") 
)

Could you please let me know why the formula is different?
Thank you!
Diana

**Comment by strengejacke** [comment]
And for logits:

``` r
esc::esc_2x2(
  grp1yes = 205,
  grp1no = 98,
  grp2yes = 512,
  grp2no = 178
)
#> 
#> Effect Size Calculation for Meta Analysis
#> 
#>      Conversion: 2x2 table (OR) to effect size logits
#>     Effect Size:  -0.3185
#>  Standard Error:   0.1505
#>        Variance:   0.0227
#>        Lower CI:  -0.6135
#>        Upper CI:  -0.0235
#>          Weight:  44.1438
```

<sup>Created on 2022-02-24 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by djmeter** [comment]
The discrepancy was in the r calculation, not OR. Thanks!

**Comment by strengejacke** [comment]
The calculator returns RR, not r.

**Comment by djmeter** [comment]
Ok, the  esc package documentation says differently, but it is helpful to know that. Thank you.
Diana

**Comment by strengejacke** [comment]
> Ok, the esc package documentation says differently

Not sure what you mean here? The function returns the effect size that is given in the `es.type` argument, which is by default `logits`. The web-based 'Practical Meta-Analysis Effect Size Calculator' returns OR, which you can also return from `esc_2x2()`, and the results are the same.

**Comment by xm0557** [comment]
I also encountered this problem.  esc_2x2(23,29,10,3, es.type="r")gave r = -0.3019; but the website showed different results.
<img width="662" alt="截屏2022-06-21 10 09 44" src="https://user-images.githubusercontent.com/45715254/174701628-fb6071f5-faa2-4e11-983c-6466017d5665.png">
